test/module/shared_model/validators/commands_validator_test.cpp

```diff
@@ -193,6 +193,20 @@ iroha::protocol::RevokePermission generateRevokePermission(
 using namespace iroha::protocol;
 using namespace shared_model;
 
+/**
+ * @given transaction without any commands
+ * @when commands validator is invoked
+ * @then answer has error about empty transaction
+ */
+TEST(commandsValidatorTest, EmptyTransactionTest) {
+  auto tx = generateEmptyTransaction();
+  tx.mutable_payload()->set_created_time(iroha::time::now());
+  shared_model::validation::CommandsValidator commands_validator;
+  auto answer = commands_validator.validate(
+          detail::make_polymorphic<proto::Transaction>(tx));
+  ASSERT_TRUE(answer.hasErrors());
+}
```

Comment:
We can also check how many errors did we get from answer. Like here https://github.com/hyperledger/iroha/pull/768/files#diff-c76782c8a0c46a09b1c21cbbe5dccb3bR332